downloads.html

```diff
@@ -16,6 +38,43 @@
   Have a look at our <a href="//wiki.minetest.net/Getting_Started">Getting Started</a>, <a href="//wiki.minetest.net/FAQ">FAQ</a> and <a href="//wiki.minetest.net/Tutorials">Tutorials</a> pages on our wiki.
 </p>
 
+<div class="intro" id="download-btn">
+    <a class="btn btn-success btn-lg" role="button">
+        Download Minetest {{ page.latest_version }}
+
+        <small>awaiting content</small>
+    </a>
+
+    <p>
+        <small>scroll down for more versions</small>
+    </p>
+</div>
+
+<script>
+    function detectPlatformName() {
+        var userAgent = navigator.userAgent.toLowerCase();
+        var platforms = ["windows", "mac", "ubuntu", "debian", "fedora"];
```

Comment:
according to https://www.whatismybrowser.com/developers/tools/user-agent-parser/browse/operating-system-name/linux-user-agents yes